### ISSUE TITLE
Внутренний совместитель.

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2337]
Originally reported on Google Code with ID 2337

```
Для того, чтобы сделать, например, приказ на изменение условий или приказ о переводе
для внутреннего совместителя мы должны изначально присваивать разные табельные номера
для работника основного производства и внутреннего совместителя. Нужно использовать,
как то возможность выбора у сотрудника либо основную либо внутреннее совмещение по
сотруднику и уже по этой должности делать нужные нам переводы.
```

Reported by `roberto1813` on 2011-02-22 15:51:16

**Comment by gsbelarus** [comment]
```
Дубликат Issue 1850
```

Reported by `Alexander.GoldenSoft` on 2011-02-22 16:00:22

**Comment by gsbelarus** [comment]
```
У меня это сделано было, но настройки гридов у нас разные...затрётся
```

Reported by `Alexander.GoldenSoft` on 2011-02-22 16:06:50

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2011-07-25 13:54:45
- **Status changed**: `Duplicate`
- **Merged into**: #1850
